packages/affine/components/src/rich-text/inline/presets/nodes/reference-node/reference-popup.ts

```diff
@@ -192,7 +193,7 @@ export class ReferencePopup extends WithDisposable(LitElement) {
             button => button.label,
             ({ label, icon, action, disabled }) => html`
               <editor-menu-action
-                ?aria-label=${label}
+                aria-label=${ifDefined(label)}
```

Comment:
Just curious. What's the difference here? why `?aria-label` won't work?

Reply:
`?aria-label` for boolean values, if it is true, the `aria-label` property will be added, but has no specific value.